app/views/organisations/_separate_website.html.erb

```diff
@@ -1,25 +1,26 @@
-<%= render "govuk_publishing_components/components/heading", {
-    text: organisation.title,
-    heading_level: 2
-} %>
-
-<div class="separate-website-label">
-  <%= "#{organisation.title} has a" %> <%= link_to "separate website", "#" %>
+<div class="grid-row">
+  <div class="column-one-third">
+    <%= render "govuk_publishing_components/components/organisation_logo", {
+      organisation: @header.organisation_logo
+    } %>
+  </div>
 </div>
 
-<div class="organisation-body">
-  <%= organisation.body %>
-</div>
+<%= render "govuk_publishing_components/components/notice", {
+  title: "#{organisation.title} has a <a href='/'>separate website</a>".html_safe
```

Comment:
I wonder if this can be translated.

Reply:
Probably. I think I'll leave this for now as I started work on this section before realising it was out of scope for this PR.

Reply:
It feels a bit weird to commit/merge half-finished work - can we cherrypick this to a new in progress branch and remove it from this one?